templates/service/build.sbt

```diff
@@ -12,21 +12,10 @@ lazy val microservice = Project(appName, file("."))
     scalaVersion                     := "$!SCALA_VERSION!$",
     libraryDependencies              ++= AppDependencies.compile ++ AppDependencies.test,
     <!--(if type=="FRONTEND")-->
-    TwirlKeys.templateImports ++= Seq(
-      "uk.gov.hmrc.$!APP_PACKAGE_NAME!$.config.AppConfig",
-      "uk.gov.hmrc.govukfrontend.views.html.components._",
-      "uk.gov.hmrc.govukfrontend.views.html.helpers._",
-      "uk.gov.hmrc.hmrcfrontend.views.html.components._",
-      "uk.gov.hmrc.hmrcfrontend.views.html.helpers._"
-    ),
     pipelineStages in Assets := Seq(gzip),
-    PlayKeys.devSettings ++= Seq("metrics.enabled" -> "false"),
```

Comment:
I can see that this has been added to application.conf instead. As @cjwebb mentioned, this risks it not being turned on again higher up. The original intent for this line was to prevent metric exceptions (e.g. `Unable to report to Graphite`) occurring when developing locally using `sbt run` only.

Reply:
`PlayKeys.devSettings` does introduce another way of doing configuration though. I think it would always supersede the value defined in application.conf? I would have to delete this line in SBT in order to see metrics getting sent whilst running locally?

I'd like this line to be removed, and then chat with others about how they handle this.

There may indeed be a better solution than we currently have, but people are already be handling this somehow. For example, some team on-boarding docs have commands to start graphite via docker. I've used Netcat as a substitute for graphite sometimes too.

Reply:
Fair enough. I'd like to see a solution to this eventually though as it is a known pain point and we shouldn't be requiring teams to have to spin up graphite or netcat to get a local build working without throwing exceptions.